product_listings_manager/tests/test_products.py

```diff
@@ -13,8 +14,8 @@
 
 
 @pytest.fixture(scope='module')
-def dbh():
-    return Products.compose_get_dbh()
+def app():
+    return create_app()
```

Comment:
It should be possible to use following to avoid `with app.app_context()` in test cases.

```python
@pytest.fixture(scope='module')
def app():
    app = create_app()
    with app.app_context():
        yield app
```

Reply:
I learned new thing. Thanks hluk.